js/index.js

```diff
@@ -1 +1,16 @@
 // Your code goes here
+
+
+var buttons = document.querySelectorAll(".btn");
+
+buttons.forEach(function(){
+	addEventListener("mouseover", function(event) {   
+    // highlight the mouseover target
+    event.target.style.color = "orange";
+
+    // reset the color after a short delay
+    setTimeout(function() {
+      event.target.style.color = "";
+    }, 500);
+  }, false);
+});
```

Comment:
It may not be much, but what you've got here is a good start! Great job for using a forEach and a setTimeout for your first one. This is awesome, but especially if you're a little behind, they don't all have to be the more complex ones.